src/drivers/gps/ashtech.cpp

```diff
@@ -100,7 +100,7 @@ int ASHTECH::handle_message(int len)
 		time_t epoch = mktime(&timeinfo);
 
 		if (epoch > GPS_EPOCH_SECS) {
-			uint64_t usecs = static_cast<uint64_t>((ashtech_sec - static_cast<uint64_t>(ashtech_sec))) * 1e6;
+			uint64_t usecs = static_cast<uint64_t>((ashtech_sec - static_cast<uint64_t>(ashtech_sec))) * 1000000;
```

Comment:
does gcc 4.9 issue a warning about this or is this just a style change? You have similar changes in the PR. Generally scientific notation is more readable. While one can argue about this with  x  * 100 it is certainly true at with x *1e6.